ping-service.tf

```diff
@@ -0,0 +1,24 @@
+module "ping_secrets" {
+  source       = "modules/service-secrets"
+  project_name = "taskcluster-ping"
+
+  secrets = {
+    DEBUG             = "*"
+    PUBLIC_URL        = "https://taskcluster.example.com/ping/"
```

Comment:
I think we don't have publicUrl anymore?  I guess ping isn't a real service so it might still use it..

Reply:
I think ping might use it. I'll try removing it though.